packages/xod-project/src/internal/patchPathUtils.js

```diff
@@ -3,7 +3,7 @@ import * as CONST from '../constants';
 
 // :: String -> Boolean
 export const isValidIdentifier = R.allPass([
-  R.test(/[a-z0-9-]+/), // only lowercase alphanumeric and hypen
+  R.test(/[a-z0-9-]*/), // only lowercase alphanumeric and hypen
```

Comment:
:exclamation: This check is used for patch paths as well. Suggest reverting and introducing additional `isProjectNameValid` on top of this check.